addons/hr_work_entry_contract/static/src/views/work_entry_calendar/work_entry_calendar_controller.js

```diff
@@ -12,6 +12,7 @@ export class WorkEntryCalendarController extends CalendarController {
         const { onRegenerateWorkEntries } = useWorkEntry({
             getEmployeeIds: this.getEmployeeIds.bind(this),
             getRange: this.model.computeRange.bind(this.model),
+            reload: this.env.searchModel.search.bind(this.env.searchModel)
```

Comment:
You should probably use `this.model.load()` instead, like it's done here: 

https://github.com/odoo/odoo/blob/fb1c37de5e7d2941bd7523ca50f274720143a7fb/addons/hr_holidays/static/src/views/calendar/calendar_controller.js#L90-L93